packages/autocertifier-client/src/data/HttpStatus.ts

```diff
@@ -0,0 +1,14 @@
+export enum HttpStatus {
```

Comment:
These are standard numbers, no need to export in `exports.ts`. The `httpStatus` of `Err` could be just a number if we need that.